aten/src/ATen/native/cpu/BlasKernel.cpp

```diff
@@ -73,7 +73,7 @@ void gemm_transa_(
     const scalar_t *b_ = b;
     for (int64_t j = 0; j < n; j++)
     {
-      scalar_t sum = 0;
+      opmath_t sum = 0;
       for(int64_t l = 0; l < k; l++)
         sum += a_[l]*b_[l];
```

Comment:
All the other operaitons in this PR happen in `opmath_t` but this one. Should at least one of the arguments be casted to `opmath_t` for the whole operation to happen in `opmath_t`?